src/utilities/integration-test-utilities.js

```diff
@@ -0,0 +1,49 @@
+import { v4 as uuidv4 } from 'uuid';
+import ModelFactory from '../models';
+import { MessageType, modelName as messageModelName } from '../models/message';
+import { logError } from '../middleware/logging';
+import { getNow } from '../services/time';
+import { Op } from 'sequelize';
+
+const Message = ModelFactory.getByName(messageModelName);
+
+export const generateRandomNhsNumber = () =>
+  Math.random().toString().slice(2, 12).replace('0', '5');
```

Comment:
if the purpose of .replace is to ensure the random number does not start with 0, maybe try this?
```suggestion
  (Math.floor(Math.random() * 9e9) + 1e9).toString();
```

Reply:
Thank you!